src/main/java/Weeny/task/Events.java

```diff
@@ -0,0 +1,62 @@
+package weeny.task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+import weeny.parser.Parser;
+/**
+ * Represents an event task with a description, start, and end date/time.
+ */
+public class Events extends Task {
```

Comment:
Should the class name be singular rather than plural since one Event is created?